packages/collapsible/src/base-types.ts

```
@@ -0,0 +1,12 @@
+import type { ContentProps as ContentPropsWeb, RootProps as RootPropsWeb } from './web/types';
+
+type BaseRootProps = Pick<
+  RootPropsWeb,
+  'open' | 'defaultOpen' | 'onOpenChange' | 'disabled' | 'asChild'
+>;
+
+type BaseTriggerProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)